Source/AdMobAdapterRewardedInterstitialAd.swift

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022-2023 Chartboost, Inc.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file.
+
+import ChartboostMediationSDK
+import Foundation
+import GoogleMobileAds
+
+final class AdMobAdapterRewardedInterstitialAd: AdMobAdapterAd, PartnerAd {
+    
+    /// The partner ad view to display inline. E.g. a banner view.
+    /// Should be nil for full-screen ads.
+    var inlineView: UIView? { nil }
+    
+    // The AdMob Ad Object
+    var ad: GADRewardedInterstitialAd?
+    
+    /// Loads an ad.
+    /// - parameter viewController: The view controller on which the ad will be presented. Needed on load for some banners.
+    /// - parameter completion: Closure to be performed once the ad has been loaded.
+    func load(with viewController: UIViewController?, completion: @escaping (Result<PartnerEventDetails, Error>) -> Void) {
+        log(.loadStarted)
+        
+        let adMobRequest = generateRequest()
+        GADRewardedInterstitialAd.load(withAdUnitID: request.partnerPlacement,
+                                       request: adMobRequest) { [weak self] ad, error in
```

Comment:
nit: can we format this call like this?
```
GADRewardedInterstitialAd.load(
    withAdUnitID: request.partnerPlacement,
    request: adMobRequest
) { [weak self] ad, error in
```

Style guide reference (https://google.github.io/swift/#line-wrapping):
![image](https://user-images.githubusercontent.com/122156786/224412287-313abe30-7253-425f-8a8e-e929524c38e7.png)